remoting/getty/config.go

```diff
@@ -179,6 +184,23 @@ func (c *GettySessionParam) CheckValidity() error {
 	return nil
 }
 
+func parseTimeDurationByRange(timeStr string, min *time.Duration, max *time.Duration) (time.Duration, error) {
+	result, err := time.ParseDuration(timeStr)
+	if err != nil {
+		return *min, err
+	}
+	if min != nil && max != nil && *min > *max { // swap
```

Comment:
The semantics of the min and max parameters are not clear,  I don't think swap is need.